opensrp-chw/src/main/java/org/smartregister/chw/util/Constants.java

```diff
@@ -178,6 +180,10 @@ public static String getChildHvMalnutritionScreening() {
 
         public static String getSkinToSkin() { return SKIN_TO_SKIN; }
 
+
+        public static String getChildHVCCDIntroduction() {
+            return CHILD_HV_CCD_INTRODUCTION;
```

Comment:
There is an error here.

Reply:
@devcolonel please fix this because there is an error and the code simply does not run.